hamilton/plugins/ibis_extensions.py

```diff
@@ -0,0 +1,75 @@
+from typing import Type
```

Comment:
this file will need to be registered in `function_modifiers/base.py` IIRC.

Reply:
@zilto are you going to add that?

Reply:
Also there should probably be a `ResultBuilder` for ibis too.

Reply:
> this file will need to be registered in `function_modifiers/base.py` IIRC.
 Why exactly? The code is working. 
 
I used this on line 71, which I find cleaner. It's not a new decorator, it's simply added to the `@check_output` supported validators
 
```python
from hamilton.data_quality import base, default_validators
# ...
def register_validators():
    default_validators.AVAILABLE_DEFAULT_VALIDATORS.append(SchemaValidatorIbis)
```

Reply:
The module registers itself based on module import, and we have a central way to do that. So while this is still needed, we shouldn't rely on this to be the way to register the module.